site/content/en/topics/tagging-resources.md

```diff
@@ -38,3 +38,18 @@ Normally the controller manager don't delete the existing tags even if they are
 | "a=b,c=d" | "" | {"e": "f"} | {"a": "b", "c": "d", "e": "f"} /* won't delete `e` because the SystemTags is empty */ |
 | "c=d" | "a" | {"a": "b"} | {"a": "b", "c": "d"} /* won't delete `a` because it's in the SystemTags */ |
 | "c=d" | "x" | {"a": "b"} | {"c": "d"} /* will delete `a` because it's not in Tags or SystemTags */ |
+
+## Including special characters in tags
+
+> This feature is supported since v1.23.0.
+
+Normally we don't support special characters such as `=` or `,` in key-value pairs. These characters will be treated as separator and will not be included in the key/value literal. To solve this problem, `tagsMap` is introduced since v1.23.0, in which a JSON-style tag is acceptable.
+
```

Comment:
Also add: Please consider migrating existing "tags" to "tagsMap", the support of "tags" configuration would be removed in a future release.

Reply:
done